app/src/main/java/org/dalol/retrofit2_restapidemo/model/helper/Constants.java

```diff
@@ -19,8 +19,8 @@
 public class Constants {
 
     public static final class HTTP {
-
         public static final String BASE_URL = "http://services.hanselandpetal.com";
+        public static final String BASE_URL_PIC = "http://services.hanselandpetal.com/photos/";
```

Comment:
here we can minimize the code duplication as public static final String BASE_URL_PIC = BASE_URL + "/photos/";